src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -103,6 +105,11 @@ protected function configure(): void
 messages didn't originate from Messenger:
 
     <info>php %command.full_name% <receiver-name> --bus=event_bus</info>
+
+Use the --queue option to limit a receiver to only certain queues. This is only possible for receivers that support restricting
```

Comment:
shouldnt the option be named `queues` if more than one are allowed?

Reply:
probably, yes. this is just a rough draft to illustrate the idea. before i get into those details, i want to know if the approach is ok or if we want to do something else.

Reply:
renamed the option